tools/test-static-code

```diff
@@ -145,4 +145,18 @@ else
     fail=1
 fi
 
+# for every C file, if there is any line starting with '#', then the first one
```

Comment:
We should strengthen this requirement, now that we are only inspecting checked in files: they should all start with `#include "config.h"` unconditionally.

Follow up please, so this can make it into the release.

Reply:
Done in PR #16786